src/action/darwin/create_fstab_entry.rs

```diff
@@ -193,7 +208,7 @@ fn fstab_entry(uuid: &Uuid, apfs_volume_label: &str) -> String {
 
 #[derive(thiserror::Error, Debug)]
 pub enum CreateFstabEntryError {
-    #[error("An `/etc/fstab` entry for the volume labelled `{0}` already exists. If a Nix Store already exists it may need to be deleted with `diskutil apfs deleteVolume \"{0}\") and should be removed from `/etc/fstab`")]
+    #[error("An `/etc/fstab` entry for the `/nix` path already exists, install will create a volume named `{0}` to mount there. If a volume named `{0}` already exists it may need to be deleted with `diskutil apfs deleteVolume \"{0}\") and the entry for `/nix` should be removed from `/etc/fstab`")]
```

Comment:
If "install will create a volume" to mount here, why is it in the error message? IMHO, we should be more clear about what actually failed here, which I think is the fact that there already exists an fstab entry, and a volume may still exist for that entry?

Would this be less wordy if we split up the errors? One error for "you already have a /nix entry from another installer. {instructions on remedying this}" and one error for "you already have a /nix entry from _our_ installer. {instructions on remedying this}"?

Reply:
It's a bit subtle since if the install creates a new volume it must also update the fstab, but this action can't know if the previous action did that (#37 ) so it can't know if the fstab is correct.

I think I can split up the errors...